### PR TITLE
extra slides into a component; extract some login qml into components and add default properties

### PR DESCRIPTION
**Pull request by iurimatias**

### CODE REVIEW

File: ui/onboarding/Login/AccountList.qml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.3
+import Qt.labs.platform 1.1
+//import "../../../../shared"
+//import "../../../../imports"
```
Comment (richard-ramos) [inline ui/onboarding/Login/AccountList.qml:6]:
```suggestion
```

File: ui/onboarding/Login/AccountSelection.qml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+import QtQuick.Controls 2.4
+import QtQuick.Layouts 1.11
+import QtQuick.Window 2.11
+import QtQuick.Dialogs 1.3
+import "../../shared"
+import "../../imports"
+
+Item {
+    property alias btnGenKey: btnGenKey
+//    property alias selectedIndex: selecte
```
Comment (richard-ramos) [inline ui/onboarding/Login/AccountSelection.qml:11]:
```suggestion
```

File: ui/onboarding/Login/AccountSelection.qml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+    id: wizardStep1
+    property int selectedIndex: 0
+    Layout.fillHeight: true
+    Layout.fillWidth: true
+
+    Text {
+        id: title
+        text: "Login"
```
Comment (richard-ramos) [inline ui/onboarding/Login/AccountSelection.qml:20]:
```suggestion
        text: qsTr("Login")
```

File: ui/onboarding/Login/AccountSelection.qml [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+        id: footer
+        width: btnGenKey.width + selectBtn.width + Theme.padding
+        height: btnGenKey.height
+        anchors.bottom: parent.bottom
+        anchors.bottomMargin: Theme.padding
+        anchors.horizontalCenter: parent.horizontalCenter
+
+        StyledButton {
+            id: btnGenKey
+            label: "Generate new account"
```
Comment (richard-ramos) [inline ui/onboarding/Login/AccountSelection.qml:46]:
```suggestion
            label: qsTr("Generate new account")
```

File: ui/onboarding/Login/AccountSelection.qml [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+        }
+
+        StyledButton {
+            id: selectBtn
+            anchors.left: btnGenKey.right
+            anchors.leftMargin: Theme.padding
+            label: "Select"
```
Comment (richard-ramos) [inline ui/onboarding/Login/AccountSelection.qml:53]:
```suggestion
            label: qsTr("Select")
```

File: ui/onboarding/Slide.qml [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+        font.bold: true
+        font.pixelSize: 22
+        font.kerning: true
+
+    }
+
+    Text {
+        id: txtDesc1
+        x: 772
+        color: "#939BA1"
```
Comment (richard-ramos) [inline ui/onboarding/Slide.qml:47]: Is this color part of the Theme?

File: ui/onboarding/Slide.qml [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+        anchors.bottomMargin: -2
+        anchors.bottom: txtDesc1.bottom
+        anchors.topMargin: -2
+        anchors.left: txtDesc1.right
+        anchors.leftMargin: 32
+        onClicked: vwOnboarding.currentIndex++
+        visible: !isLast
+        background: Rectangle {
+            id: rctNext1
+            color: "#ECEFFC"
```
Comment (richard-ramos) [inline ui/onboarding/Slide.qml:73]: Same here?

File: ui/onboarding/Login/AccountSelection.qml (hunk 4 above)
Comment (iurimatias) [reply ui/onboarding/Login/AccountSelection.qml:46]: if the property is for GenKey.qml, then will do in a different PR when dealing with GenKey

**Pull request merged**